lib/generators/shopify_app/install/templates/flash_messages.js

```diff
@@ -4,12 +4,21 @@ if (!document.documentElement.hasAttribute("data-turbolinks-preview")) {
   document.addEventListener(eventName, function flash() {
     var flashData = JSON.parse(document.getElementById('shopify-app-flash').dataset.flash);
```

Comment:
Can we remove all instances of `flash` and rename to `toast` for consistency?

Reply:
Yup, I'll update where it makes sense. Some of these will stay named `flash` just because that's the idiom [Rails uses](https://api.rubyonrails.org/classes/ActionDispatch/Flash.html)